pkg/reconciler/revision/cruds.go

```diff
@@ -55,7 +55,8 @@ func (c *Reconciler) createSecret(ctx context.Context, ns *corev1.Namespace) (*c
 			Namespace:       ns.Name,
 			OwnerReferences: []metav1.OwnerReference{*metav1.NewControllerRef(ns, corev1.SchemeGroupVersion.WithKind("Namespace"))},
 			Labels: map[string]string{
-				networking.ServingCertName + "-ctrl": "data-plane",
+				networking.ServingCertName + "-ctrl":              "data-plane",
+				"networking.internal.knative.dev/certificate-uid": networking.ServingCertName,
```

Comment:
This label is not necessary for now. In the future, we **may** need this when we start using the certificate signed by each CA, but this secret is not read by Ingress so far.

Reply:
I had this here for the sake of completeness of the PR.

Reply:
Sorry what do you mean for the "completeness of the PR."?

Reply:
Yeah, we should add it in the future when it is necessary.